kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -4,7 +4,7 @@
  */
 
 const vue = require('vue');
-const VueIntl = require('vue-intl');
+// const VueIntl = require('vue-intl');
```

Comment:
I was getting build issues when I was working on the 6/1, I saw the commit that updated vue-intl. If this has been fixed I'll make another commit bringing it back in.